Description: use .html extension for HTML documentation files
 Define %html-ext% to ".html" while building HTML documentation, since current
 docbook-dsssl appears to default to ".htm".
Author: Matthias Klose <doko@debian.org>
Forwarded: no
Last-Update: 2013-10-30

--- a/Docs/sgmlmanual/HTML.dsl
+++ b/Docs/sgmlmanual/HTML.dsl
@@ -10,7 +10,7 @@
 ;; your stuff goes here...
 ;;(define %gentext-nav-use-ff% 1)
 (define %use-id-as-filename% 1)
-
+(define %html-ext% ".html")
 
 </style-specification-body>
 </style-specification>
